daemon/daemon_unix.go

```diff
@@ -824,11 +823,9 @@ func parseRemappedRoot(usergrp string) (string, string, error) {
 			groupname = lgrp.Name
 		} else {
 			// not a number; attempt a lookup
-			group, err := user.LookupGroup(idparts[1])
-			if err != nil {
+			if _, err := user.LookupGroup(idparts[1]); err != nil {
 				return "", "", fmt.Errorf("Error during gid lookup for %q: %v", idparts[1], err)
 			}
-			groupID = group.Gid
```

Comment:
Yes, because if the group doesn't exist in `/etc/group` then the whole function should error out. The error text should be updated--should be "Error during **groupname** lookup..", not **gid**.